src/main/webapp/creatives/maps/utils.js

```diff
@@ -0,0 +1,17 @@
+
+
+/**
+ * Sets the session ID, this should only be used on session creation.
```

Comment:
This doesn't really set the session id.  You could change the definition to just have the return param

/**
 * @return {string} A statndard UUID conforming to en.wikipedia.org/wiki/Universally_unique_identifier
 */